app/models/job.ts

```diff
@@ -407,6 +408,24 @@ export class Job extends Record {
     return terminalStates.includes(this.status);
   }
 
+  /**
+   * Check if the job has any STAC data links
+   *
+   * @param transaction - transaction to use for the query
+   * @returns true or false
+   */
+  async hasStacLinks(
```

Comment:
Similar comments to `getLinksForJob`, I think this bleeding dependencies on presentation into the model a little too much.

If you implement the changes suggested there, I'd carry them over here as params.  `hasLinks(rel=null, requireSpatioTemporal=false)` or similar.  Then the method body could and probably should just call `getLinksForJob` setting `perPage=1`, which amounts to the same query but means that the query logic exists in one method rather than 2, making it easier to update in the future.

Reply:
Agreed, thanks for the suggestions. I'll probably break it into multiple params rather than adding a new method.